CONTRIBUTING.md

```diff
@@ -83,12 +83,3 @@ When reviewing your contribution, we apply the following criteria:
 
 TODO: Should we include a Code of Conduct, like
 https://github.com/discourse/discourse/blob/master/docs/code-of-conduct.md ?
-
-## ThoughtWork's role
-
-ThoughtWorks seeds the community that builds Coy IM. We seed the development, investing our own resources: We provide a team of software delivery experts to lay the foundation for the project. We use our network and contacts to approach customers and users.
-
-ThoughtWorks started building Coy IM because it is right. In combining our passion for defending a free internet and our capability to deliver software, we build software to counter widespread mass surveillance of email communication.
-
-It is not ThoughWork's goal to make money from Coy IM. The reasons are multiple, but at the end of the day, we believe that our goals of ensuring digital privacy and anonymity for every person on the planet can best be achieved if Coy IM puts end users before a revenue stream.
-
```

Comment:
Hey,

Thanks for this!!

On the chat, was discussed with @olabini that maybe is better to mention TW's role in the previous development.. wanna change that?

Reply:
hey @claucece, that's a good call! Thanks for the feedback.
I'm gonna need some help to explain TW -> CAD transition.